test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,68 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.commands.HelpCommand;
+import seedu.addressbook.data.exception.IllegalValueException;
+
+
+public class NameTest {
+    
+    /*
+     * Tests for the isSimilar method
+     */
+
+    @Test
+    public void checkifSimilarTestWithEqual() throws IllegalValueException {
```

Comment:
the test name should have 3 parts `<method under test>_<which scenario/test case>_<expected return value>`